recipes/opensim/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+    name: opensim
+    version: "4.3.1"
+
+source:
+    git_url: https://github.com/opensim-org/opensim-core.git
+    git_rev: "4fb99b9d6b8e779601c45a63b3557780c432f1f8"
```

Comment:
This should source from a release tarball instead of git.

Reply:
Thanks @moorepants I'll close this PR and resurrect the other PR started earlier by Chris